manifest.json

```diff
@@ -480,10 +480,213 @@
       "version": "0.1"
     },
     {
-      "description": "formatters for various languages",
+      "description": "Formatters for various languages.",
       "id": "formatter",
       "mod_version": "3",
-      "remote": "https://github.com/vincens2005/lite-formatters:7e017080a967c01d437e484247a90c1ff52e8ef8",
+      "remote": "https://github.com/vincens2005/lite-formatters:e5d0ccbad76a51969f9dc8392a33b15c06ee84b8",
+      "version": "0.2"
```

Comment:
This will have to wait until the remote manifest version gets updated to `0.2` too.

Reply:
It's been updated.